### DISCUSSION TITLE
Black screen when launching installer.elf

### DISCUSSION THREAD

**Discussion by L0g1calMadn3ss** [israpps/FreeMcBoot-Installer#26] [Q&A] [answered]
@israpps  FMCB 1.966 installer elf from noobie package works (ish, it freezes on actually installing FMCB on official MCs, sometimes...) but I get a black screen when launching this 1.966 version.  Tried a lot of little things to get it working and I'm stumped. *Edit, warranty sticker is still on the SCPH-30001 console. TIA

**Comment by L0g1calMadn3ss** [comment, marked as answer]
The version of uLe/wLe you use to launch the installer is what causes the "black screen".  I booted FMCB, then browsed through the "INSTALL" folder of your installer, launched the BOOT.ELF within, and used that to launch the FMCBInstaller.elf. Currently reinstalling to all my MCs and HDDs. Very happy to be using Israpps stuff! :D

**Reply by TorqueEmPup** [reply]
This made no difference for me, launching the boot.elf in the install folder just reloads the Elf Launcher.

**Reply by israpps** [reply]
> This made no difference for me, launching the boot.elf in the install folder just reloads the Elf Launcher.

let me get this right. when you run the installer it kicks back to main menu of console?

**Comment by Ska044** [comment]
> @israpps نصب‌کننده FMCB 1.966 از بسته noobie به اسم elf کار می‌کند (البته، موقع نصب FMCB روی MC های رسمی هنگ می‌کند، گاهی اوقات...) اما موقع اجرای این نسخه ۱.۹۶۶ با صفحه سیاه مواجه می‌شوم. کلی کار کوچک انجام دادم تا درست شود و گیر کرده‌ام. *ویرایش، برچسب گارانتی هنوز روی کنسول SCPH-30001 است. TIA

Hello I also have this problem, if you solve it, tell me 🙏❤️
